packages/bits-ui/package.json

```
@@ -58,6 +58,6 @@
 		"nanoid": "^5.0.5"
 	},
 	"peerDependencies": {
-		"svelte": "^4.0.0"
+		"svelte": "^4.0.0 || ^5.0.0"
```

`^5.0.0` works with pnpm, but not with npm. We must use `-next.X`
```suggestion
		"svelte": "^4.0.0 || ^5.0.0-next.118"
```